src/conf_mode/containers.py

```diff
@@ -206,14 +207,35 @@ def apply(container):
                       tmp += ' --ipv6'
                 _cmd(tmp)
 
+
+        # disable masquerading and use traditional bridging so VyOS can control firewalling/NAT/etc on the cni-podmanX interface
+        for network, network_config in container['network'].items():
```

Comment:
This is a separate feature compared to the Dockerfile and should be shipped in an individual commit.

Reply:
Will move to a different task and commit